lib/MetaCPAN/Server/Controller/Changes.pm

```diff
@@ -65,4 +65,41 @@ sub all : Chained('index') : PathPart('') : Args(0) {
     $c->detach('not_found');
 }
 
+sub by_releases : Path('by_releases') : Args(0) {
+    my ( $self, $c ) = @_;
+
+    # ArrayRef[ Dict[ author => Str, name => Str ] ]
+    my $arg = $c->read_param("releases");
+    my $ret = $c->model('CPAN::Release')->by_author_and_names($arg);
+
+    my @changes;
+    for my $release ( @{ $ret->{releases} } ) {
+        my ( $author, $name, $path )
+            = @{$release}{qw(author name changes_file)};
+        my $source = $c->model('Source')->path( $author, $name, $path ) // '';
```

Comment:
This should either be `// next;` or the `// ''` should be removed and the later code should avoid trying to read the file.